Source/WebCore/style/StyleAdjuster.cpp

```diff
@@ -696,7 +696,9 @@ void Adjuster::adjust(RenderStyle& style, const RenderStyle* userAgentAppearance
             || style.hasIsolation()
             || style.hasMask()
             || style.hasBackdropFilter()
-            || style.hasBlendMode();
+            || style.hasBlendMode()
+            || style.viewTransitionName()
+            || (m_element && m_element->capturedInViewTransition());
```

Comment:
Maybe add a comment to say this is in sync with `requiresRenderingConsolidationForViewTransition`